hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1486,7 +1484,9 @@ private static UserGroupInformation getRemoteUser() throws IOException {
         realUser = new Text(ugi.getRealUser().getUserName());
       }
 
-      return delegationTokenMgr.createToken(owner, renewer, realUser);
+      token = delegationTokenMgr.createToken(owner, renewer, realUser);
+      LOG.debug("OmDelegationToken: {} created.", token);
```

Comment:
We already have a trace statement inside SecretManager. Shall we skip this debug statement?

Reply:
Remove this one and change the trace log to debug log in SecretManager.